### PR TITLE
Translate  a contribute to French

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: content/fr/docs/contribute/style/write-new-topic.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+Choisissez un titre qui contient les mots-clés que vous voulez que les moteurs de recherche trouvent.
+Créez un nom de fichier qui utilise les mots de votre titre séparés par des tirets.
+Par exemple, le sujet avec titre [Using an HTTP Proxy to Access the Kubernetes API](/docs/tasks/access-kubernetes-api/http-proxy-access-api/) has filename `http-proxy-access-api.md`.
+Vous n'avez pas besoin de mettre "kubernetes" dans le nom du fichier, car "kubernetes" est déjà dans l'URL du sujet, par exemple :
+
+       /docs/tasks/access-kubernetes-api/http-proxy-access-api/
+
+## Ajout du titre du sujet à l'entête
+
+In your topic, put a `title` field in the [front matter](https://jekyllrb.com/docs/frontmatter/).
```
Comment (AWKIF) [inline content/fr/docs/contribute/style/write-new-topic.md]: A traduire

File: content/fr/docs/contribute/style/hugo-shortcodes/index.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+## Glossaire
+
+Vous pouvez faire référence à des termes du glossaire avec une inclusion qui met à jour et remplace automatiquement le contenu avec les liens pertinents de [notre glossaire](/fr/docs/reference/glossary/).
+When the term is moused-over by someone using the online documentation, the glossary entry displays a tooltip.
+
+The raw data for glossary terms is stored at [https://github.com/kubernetes/website/tree/master/content/en/docs/reference/glossary](https://github.com/kubernetes/website/tree/master/content/en/docs/reference/glossary), with a content file for each glossary term.
+
+### Démonstration du glossaire
+
+Par exemple, l'include suivant dans les rendus de démarque est rendu à {{< glossary_tooltip text="cluster" term_id="cluster" >}} avec une infobulle :
```
Comment (AWKIF) [inline content/fr/docs/contribute/style/hugo-shortcodes/index.md]: voir si on garde "l'include"

**Comment by remyleone** [comment]
@AWKIF fixed

**Pull request merged**